modules/06 Angular/Assignments/12/answer12/src/app/image/image.component.html

```diff
@@ -0,0 +1,6 @@
+<div *ngFor="let image of images">
```

Comment:
Why is this a kind of list of images? I'm seeing one image in the name of the component